packages/frontend/src/components/UserSettings/UsersAndGroupsPanel/GroupsView.tsx

```diff
@@ -168,7 +168,7 @@ const GroupsView: FC = () => {
     const [search, setSearch] = useState('');
 
     const { data: groups, isInitialLoading: isLoadingGroups } =
-        useOrganizationGroups({ search, includeMembers: 100 }); // TODO: pagination
+        useOrganizationGroups({ search, includeMembers: 2000 }); // TODO: pagination
```

Comment:
non-blocking - we should have this as a constant here

Reply:
Yea, I thought about that, but we don't really have any kind of constants location or management. Do you mean just at the top of the file? I can at least do that.